zeroeq/http/server.cpp

```diff
@@ -285,48 +328,77 @@ class Server::Impl : public detail::Sender
 protected:
     typedef std::map< std::string, PUTPayloadFunc > PUTFuncMap;
     typedef std::map< std::string, GETFunc > GETFuncMap;
-    PUTFuncMap _subscriptions;
-    GETFuncMap _registrations;
+    typedef std::map< std::string, std::string > SchemaMap;
+    PUTFuncMap _put;
+    GETFuncMap _get;
+    SchemaMap _schemas;
 
     std::string _getTypeName( const std::string& url )
     {
         if( url.empty( ))
             return url;
 
-        std::string name = url.substr( 1 );
-        while( true )
+        return _replaceAll( url.substr( 1 ), "/", "::" );
+    }
+
+    std::string _returnRegistry() const
+    {
+        Json::Value body( Json::objectValue );
+        for( const auto& i : _get )
         {
-            const size_t pos = name.find( '/' );
-            if( pos == std::string::npos )
-                return name;
+            const std::string key = _replaceAll( i.first, "::", "/" );
```

Comment:
wouldn't it be more logical to store the keys with '/' directly? Only the servus::Serializable need a conversion at insertion time, no?